queries.sql

```diff
@@ -1,3 +1,66 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = TRUE;
+
+SELECT * FROM animals WHERE name <> 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = 'null';
+
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+BEGIN;
+SAVEPOINT my_savepoint;
```

Comment:
[OPTIONAL]
- [ ] According to the project requirement, we were supposed to:
  - Delete all animals born after Jan 1st, 2022
  - Create a savepoint for the transaction
  - Update all animals' weight to be their weight multiplied by -1.
  - Rollback to the savepoint
  - Update all animals' weights that are negative to be their weight multiplied by -1.
  - Commit transaction
  All in the same transaction. 
![Screenshot from 2023-09-19 19-42-00](https://github.com/obibaadoma/Vet_clinic_database/assets/91142432/ba1e688f-7513-407d-b260-0c40ea9c6182)

So your transaction could look like this :point_down: 
```sql
    BEGIN;

    DELETE FROM animals WHERE  date_of_birth > '2022/01/01' ;
    SAVEPOINT SP1;

    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 

    COMMIT;
```

_**This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you make the change**_

Reply:
updated code